torchvision/transforms/functional_pil.py

```diff
@@ -249,7 +249,7 @@ def resize(
     if not (isinstance(size, list) and len(size) == 2):
         raise TypeError(f"Got inappropriate size arg: {size}")
 
-    return img.resize(size[::-1], interpolation)
+    return img.resize(tuple(size[::-1]), interpolation)
```

Comment:
From the error message
```
SystemError: new style getargs format but argument is not a tuple
```
seems like the accimage resize only accept tuple, hence we convert to tuple here.
I have tested and it seems to work. However I am not really sure this is the best fix.